verible/verilog/analysis/descriptions.h

```diff
@@ -35,14 +35,16 @@ struct LintConfigParameterDescriptor {
   std::string description;
 };
 
-struct LintRuleDescriptor {
```

Comment:
where did this one go ? Looks like the end of the file was accidentally deleted ?